src/cppgenerator/Header.js

```diff
@@ -19,7 +19,7 @@ function genBuilder(widget) {
   let res = `${id}(
             UI.${type.toLowerCase()}(`
 
-  const coordNames = ['x', 'y', 'w', 'h']
+  const coordNames = ['x', 'y', 'w', 'h', 'tab']
   res += coordNames.map((n) => Common.getPropertyValue(widget, n)).join(', ')
```

Comment:
Bude to generovat `.hpp` nekompatibilní se starším GridUi, takže je třeba zvýšit `MIN_LIBRARY_VERSION` ve `05_widget.js`. Taky by bylo super, kdyby to umělo vygenerovat i hlavičku kompatibilní se starší verzí, ale klidně to přidám já.